File: app_java/src/main/java/vlab/android/architecture/base/BaseUseCase.java
package vlab.android.architecture.base;

import vlab.android.common.util.LogUtils;

/**
 * Created by Vinh.Tran on 10/25/18.
 **/
public class BaseUseCase {
    /**
     * clear resource hear
     */
    protected void onCleared(){
        LogUtils.d(getClass().getSimpleName(), ">>> onCleared ");
    }
}

File: app_kotlin/src/main/java/vlab/android/architecture/base/BaseUseCase.kt
package vlab.android.architecture.base

import vlab.android.common.util.LogUtils

/**
 * Created by Vinh.Tran on 10/25/18.
 */
open class BaseUseCase {
    /**
     * clear resource hear
     */
    protected open fun onCleared() {
        LogUtils.d(javaClass.simpleName, ">>> onCleared ")
    }
}
